clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/resources/servergroup/ops/ResizeAzureServerGroupAtomicOperation.groovy

```diff
@@ -0,0 +1,100 @@
+/*
```

Comment:
We'd prefer newly-added files to not be authored in groovy. The java files can still live in the same directories as the pre-existing groovy files.

Reply:
Thanks for your reminder. As @mybayern1974 said, is it ok to merge it now given one of our customers only use the Red/Black strategy so we'd like to unblock them asap. Meanwhile we will start the work of converting these newly added files to react from now on, and please expect our next PR to have it.